closed/src/java.base/share/classes/sun/security/ec/NativeECDHKeyAgreement.java

```diff
@@ -120,6 +127,15 @@ private void init(Key key)
             ECUtil.checkPrivateKey(keyImpl);
 
             this.privateKey = keyImpl;
+            nativePrivateKey = NativeECUtil.getPrivateKeyNativePtr(this.privateKey);
+            if (nativePrivateKey == -1) {
```

Comment:
Please use the local rather than the field, and qualify field references with `this.`.

Reply:
Changed.